### DISCUSSION TITLE
[Pathway Discussion]: WP5424 - How to represent the link to senescence?

### DISCUSSION THREAD

**Discussion by JuliaBatUM** [wikipathways/wikipathways-help#81] [pathway-discussion]
### Discussion

Hi!
The project goal is to create a map containing oxylipins and their role in senescence. For that, I tried to visually highlight enzymes, reactions, or metabolites that are changed in senescence. It is also important to know that there are multiple types of senescence, such as:

- Irradiation-induced senescence
- Oncogene-induced senescence (oncogene)
- DNA damage-induced senescence
- Oxidative stress-induced senescence
- etc. 

These different types of senescence do not necessarily induce the same phenotype in senescent cells, but some may overlap (eg, PTGES-PTGES2-PTGES3). 
I always represented the type of senescence in pink. I did it mainly for me to have it visually clearer, especially because I am not yet finished adding possible links to senescence. So I am open to suggestions on how to represent it clearer. 
When senescence X upregulates a gene (or enzyme) or leads to the overproduction of a metabolite, I represented this "interaction" as a stimulation arrow in green (eg, radiation-induced senescence and oncogene-induced senescence lead both to the upregulation of the enzyme PTGES2). In contrast, when senescence X downregulates a gene or there is a decrease in the metabolite production, I represented this "interaction" as a stimulation arrow in red (eg, radiation-induced senescence and oncogene-induced senescence lead both to the downregulation of the enzyme CYP8A1). 
Right now, I am representing it like that, as I am hoping to get a clearer image about what is exactly up/downregulated in senescence as some articles do not necessarily investigate both on the protein level and gene level. 
![image](https://github.com/wikipathways/wikipathways-help/assets/130973169/9fcffce5-492f-4152-8a3b-1529c4876e30)

During the WikiPathway meeting, we discussed to:

- use the **same name and ID** for each type of senescence ->I will start using consistent terms
- use **WikiData/Ontology tags** to identify each type of senescence -> I am not sure how to do that. For the whole pathway, I now added an ontology tag termed "_cellular senescence pathway_", but this is more a general name including all types of senescence. I also don´t know how to add ontology tags directly into the map specifically for each type of senescence
- use **Orphanet ID** to identify each type of senescence -> I visited the website but I could not find anything about senescence. I think Orphanet is more used for disease IDs and I suspect senescence to be not seen as a disease. 
- create an “**unknown” data type** for adding the different types of senescence and add in the comment a term that can be easily recognized and is consistent-> I will think use the unknown data type as it seems to be the best and easiest solution right now

Thank you very much for your help. I hope everything is clear,
Julia

### Type of revision

- [ ] Major
- [ ] Minor
- [ ] Deletion

**Comment by khanspers** [comment]
Thanks @JuliaBatUM. Some comments below:

`- use **WikiData/Ontology tags** to identify each type of senescence -> I am not sure how to do that. For the whole pathway, I now added an ontology tag termed "_cellular senescence pathway_", but this is more a general name including all types of senescence. I also don´t know how to add ontology tags directly into the map specifically for each type of senescence`
=> I think we can only use WikiData identifiers for metabolites in PathVisio (@egonw please correct me if I'm wrong), but in general the idea was probably to search [WikiData](https://www.wikidata.org/wiki/Wikidata:Main_Page) for the various types of senescence, and then add the identifier manually for that node in PathVisio, or at least as a comment.
For Gene Ontology terms, they can only be added as a comment in the current version of PathVisio.  

`- create an “**unknown” data type** for adding the different types of senescence and add in the comment a term that can be easily recognized and is consistent-> I will think use the unknown data type as it seems to be the best and easiest solution right now`
=> This is a good idea, but if an identifier can't be added to the node I think a simple label would work just as well. You can add comments on labels as well.

**Reply by JuliaBatUM** [reply]
@khanspers Thanks for the clarifications.

Would it then be better to use a simple label ? Or can I continue using the unknown data type?

Kind regards,
Julia

**Reply by khanspers** [reply]
@JuliaBatUM: I reviewed this again, as well as the notes from the meeting. If some of the items (i.e. different types of senescence) will be associated with identifiers at some point, then they all instances of senescence should be of a type that supports identifiers, so not a simple label. However, Im not sure I understand the use of the Unknown data type here; it seems to me that a Pathway node would fit better, since each type of senescence describes a process, right? Since I missed the meeting I might be missing some important context about the decision to use the Unknown data node. 

Along the same lines, there are other nodes in the pathway that probably have the wrong type. For example "Beta-oxidation" should be a pathway node, "Lipoxins" should be a metabolite, "Leukotriene 3-series" should be a metabolite, "Non-enzymatic Hydrolysis" should be a pathway node etc.
